client/src/components/containers/SetCourseForm.tsx

```diff
@@ -83,32 +98,52 @@ function SetCourseForm() {
     dispatch(setPosition({ lat: 0, lng: 0 }));
     dispatch(setMarker(''));
   };
-  const driverIds = ['5fc4aab0aa5f0a79191c2bd5', '2', '3'];
+
   const handelCourseSubmitButton = () => {
-    callRequest(client, driverIds, 'riderId', originPosition, destPosition);
+    const riderPublishInfo: riderPublishInfo = {
```

Comment:
```suggestion
    const riderPublishInfo: RiderPublishInfo = {
```
or
```suggestion
    const riderPublishInfo: RiderPublishInfoInterface = {
```
여기도 같은 이유로 바꿔주면 좋을 것 같습니다.